modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc

```diff
@@ -49,3 +49,8 @@ All DNS records must be subdomains of this base domain and include the cluster n
 . Download the discovery ISO.
 
 . Make a note of the discovery ISO URL for installing with virtual media.
+
+[NOTE]
+=====
+If you enable OpenShift virtualization during this process, you must have a second local storage device of at least `50GB` for your virtual machines.
```

Comment:
Based on this list: https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc

You might wanna run that by someone who works with CNV/try it and make sure it does in fact render "OpenShift Virtualization" as I believe it does. 

```suggestion
If you enable {VirtProductName} during this process, you must have a second local storage device of at least 50 GB for your virtual machines.
```